wordpress-mobile/WordPress-Android
# [Jetpack Full Plugin] Install flow UI tweaks

Fixes #18005 

This PR implements the tweaks proposed by the design team here: qk5s4q4mZsXknwhMWo2vIY-fi-9811%3A21323

## Demos
<details>
<summary>"Jetpack Full Plugin install" flow with tweaks - Success</summary>

https://user-images.githubusercontent.com/5091503/222804273-1bb7ae8d-7465-4600-8b0e-2518f7c88691.mp4

</details>

<details>
<summary>"Jetpack Full Plugin install" flow with tweaks - Error</summary>

https://user-images.githubusercontent.com/5091503/222804302-816ff84f-a1f6-4ad7-904c-5174267c9c4d.mp4

</details>

## To test
Set up a self-hosted site that has only individual Jetpack plugins enabled and connect it to a WP.com account.

1. Install Jetpack app
2. Log in with the same WP.com account
2. Restart the app (so the site appears in the site list)
2. Select the self-hosted site from the site list
2. The dialog asking to install the full plugin should appear AND/OR the card on the Dashboard
2. **Testing the error state**
    1. Turn on Airplane mode
    2. Start the Full Plugin Install Flow
    2. **Verify** the UI is updated according to expectations
2. **Testing the install flow**
    1. Close the Full Plugin Install flow
    2. Turn off Airplane mode
    2. Start the Full Plugin Install Flow
    2. **Verify** the UI is updated according to expectations
    3. **Verify** everything worked and the Jetpack plugin was installed

## Regression Notes
1. Potential unintended areas of impact
Behavior / installation issues.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

5. What automated tests I added (or what prevented me from doing so)
Updated existing tests only, most changes were just on UI.

PR submission checklist:
